spec/features/project/invitations_spec.rb

```diff
@@ -96,4 +96,26 @@
     expect(@project.hiwis.size).to eq 1
     expect(chair.hiwis.include?(@user)).to be true
   end
+
+  it 'can re-invite a user after he declined' do
+    chair = FactoryGirl.create(:chair, name: 'Test Chair')
+    user1 = FactoryGirl.create(:user)
+    representative1 = FactoryGirl.create(:chair_representative, user: user1, chair: chair).user
+    @project.update(chair: representative1.chair)
+    chair.projects << @project
+    expect(chair.hiwis.size).to eq 0
+
+    invitation = FactoryGirl.create(:invitation, user: @user, project: @project, sender: @user)
+    FactoryGirl.create(:event_project_invitation, trigger: invitation, target: @user, seclevel: :hiwi, type: 'EventProjectInvitation')
+    visit '/dashboard'
+    click_on 'Decline'
```

Comment:
Ich sehe darin keinerlei Vorteile gegenüber der expliziten Variante:
- die Tests werden standardmäßig immer in English ausgeführt (was jeder Entwickler können sollte ;) )
- und das Einfügen von I18n-keys in den Test macht ihn meines Erachtens nur unleserlicher. Wenn ich einen Test sehe, möchte ich sofort wissen, was er macht.